ShopifySharp/Entities/Refund.cs

```diff
@@ -59,12 +59,12 @@ public class Refund : ShopifyObject
         /// </summary>
         [JsonProperty("refund_line_items")]
         public IEnumerable<RefundLineItem> RefundLineItems { get; set; }
-
+        
         /// <summary>
-        /// Whether or not the line items were added back to the store inventory.
+        /// How this refund affects inventory levels.
         /// </summary>
-        [JsonProperty("restock")]
-        public bool? Restock { get; set; }
```

Comment:
While this `Restock` property is no longer valid, let's leave it in and just decorate it with an obsolete attribute which will let everyone who's using it know they should use `RestockType` instead. Then I'll remove the `Restock` property in 5.0.

So something like this:

```cs
/// <summary>
/// Previously this property would indicate whether the line items were added or should be added back to the store. It is now obsolete and ignored by Shopify.
/// </summary>
[JsonProperty("restock")]
[Obsolete("The Restock property is now obsolete and will be removed in a future release. Attempting to use this property when creating or updating a Refund will result in the Shopify API returning an error.")]
public bool? Restock { get; set; }
```